### PR TITLE
chore: release react-icons-compat package

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR releases the `react-icons-compat` package that can be used to access the v8 `initializeIcons` and `registerIcons` api without relying on any v8 packages.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-icons-compat-3e8c3809-167b-430e-8a34-de50112f99de.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "'feat: release react-icons-compat to stable'",
```
Comment (sopranopillow) [inline change/@fluentui-react-icons-compat-3e8c3809-167b-430e-8a34-de50112f99de.json:3]:
```suggestion
  "comment": "feat: Release react-icons-compat to stable",
```

**Review by sopranopillow — COMMENTED** [review]
qq, was this done through the generator, or did you manually added the changes? I think there are a few changes missed documentation related, like removing unstable warnings.

File: change/@fluentui-react-icons-compat-3e8c3809-167b-430e-8a34-de50112f99de.json (hunk 1 above)
Comment (tomi-msft) [reply change/@fluentui-react-icons-compat-3e8c3809-167b-430e-8a34-de50112f99de.json:3]: This was done through the script, I guess it doesn't change much when the phase is set to `compat` vs when its set to `stable`

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**